src/components/index.js

```diff
@@ -11,4 +11,5 @@ Vue.component('font-awesome-icon', FontAwesomeIcon)
 Vue.component('font-awesome-layers', FontAwesomeLayers)
 Vue.component('c-permissions-button', components.CPermissionsButton)
 Vue.component('c-input-confirm', components.CInputConfirm)
+Vue.component('c-item-picker', components.CItemPicker)
```

Comment:
I don't see any benefits in importing item picker as global component. And you are already importing it in the wrapping component, right?

Reply:
you're right. Removing it